src/GameWorld.cpp

```diff
@@ -321,4 +325,12 @@ void GameWorld::updateInternalGameObjectLists() {
 		staticGameObjects_.push_back(staticGameObjectsToAdd_.front());
 		staticGameObjectsToAdd_.pop();
 	}
+
+	while (!dynamicGameObjectsToRemove_.empty()) {

```

Comment:
You'll probably have to fix this since I'm assuming this is where the merge conflict will be.